pymemcache/serde.py

```diff
@@ -30,6 +30,14 @@
 FLAG_COMPRESSED = 1 << 3  # unused, to main compatability with python-memcached
 FLAG_TEXT = 1 << 4
 
+# Pickle protocol version (-1 for highest available to runtime)
+# Warning with `0`: If somewhere in your value lies a slotted object,
+# ie defines `__slots__`, even if you do not include it in your pickleable
+# state via `__getstate__`, python will complain with something like:
+#   TypeError: a class that defines __slots__ without defining __getstate__
+#   cannot be pickled
+PICKLE_VERSION = -1
```

Comment:
Why not use https://docs.python.org/2/library/pickle.html#pickle.HIGHEST_PROTOCOL?

Also note that in python3 the default is not 0.  https://docs.python.org/3.1/library/pickle.html